src/PageContent/Calendar/EventCard/EventCard.js

```diff
@@ -22,13 +23,15 @@ class EventCard extends Component {
             truckAlt = 'grey truck';
             style = 'event-container event-container-past';
         }
-
+        let startTime = this.props.delivery[0].startTimestamp;
```

Comment:
shouldn't the event passed in `this.props.events[e]` be a single `delivery` object? why indexing by 0? same with everywhere else. Each `EventCard` should only deal with 1 delivery